### PR TITLE
created some additionl pages for all models

### PR DESCRIPTION
**Pull request by Timur5050**

### CODE REVIEW

**Comment by Timur5050** [comment]
image
image
image
image
image

image
image

image

File: taxi/views.py [hunk 1]
```diff
@@ -13,3 +15,32 @@ def index(request):
...
+
+
+class DriverListView(ListView):
+    model = Driver
+    paginate_by = 5
+
+
+class DriverDetailView(DetailView):
+    model = Driver
+    queryset = Driver.objects.all().prefetch_related(
```
Comment (pavlopro) [inline taxi/views.py]: Simplify to

`Driver.objects.prefetch_related("cars__manufacturer")`

File: taxi/views.py [hunk 2]
```diff
@@ -13,3 +15,32 @@ def index(request):
     }
 
     return render(request, "taxi/index.html", context=context)
+
+
+class ManufacturerListView(ListView):
+    model = Manufacturer
+    queryset = Manufacturer.objects.all().order_by("name")
```
Comment (pavlopro) [inline taxi/views.py]: You can omit `all()` here

File: taxi/views.py [hunk 3]
```diff
@@ -13,3 +15,32 @@ def index(request):
...
+    paginate_by = 5
+
+
+class CarListView(ListView):
+    model = Car
+    queryset = Car.objects.all().select_related("manufacturer")
```
Comment (pavlopro) [inline taxi/views.py]: You can omit `all()` here

**Review by pavlopro — CHANGES_REQUESTED** [review]

**Review by pavlopro — APPROVED** [review]
Good Job!

**Pull request closed without merging**